### ISSUE TITLE
[Help]: How to calculate the distance between two devices?

### ISSUE THREAD

**Issue by luan-coelho** [boskokg/flutter_blue_plus#687]
### Requirements

- [X] I've looked at the README 'Common Problems' section

### Have you checked this problem on the example app?

No

### FlutterBluePlus Version

1.28.14

### Flutter Version

3.16.0

### What OS?

All

### OS Version

latest versions

### Bluetooth Module

I don't know how to inform

### What is your problem?

I'm working on developing a mobile application to control student and teacher attendance. At a certain point it is necessary to validate the student's presence by checking the distance between their device and the teacher's device. I would like to know if the current library offers adequate resources to perform this distance calculation between devices. Any guidance or recommendations on methods or best practices for this implementation would be greatly appreciated.

In summary, does this library have enough features to calculate the distance between devices?

### Logs

```shell
I don't know how to inform
```

**Comment by chipweinberger** [comment]
sure.

scan and use `scanResult..rssi`
